### DISCUSSION TITLE
What is the mathematical base for the PK-Sim model simulation?

### DISCUSSION THREAD

**Discussion by xutt0913** [Open-Systems-Pharmacology/Forum#1352] [Models]
I spent a long time but did not find any information about the mathematical base of this software. Could anyone explain to me? Is it Bayesian method based?

**Comment by tobiasK2001** [comment]
Hm, it is probably possible to apply Bayesian methods for certain applications. But if you want to find out more about the mathematics (I think ODE based) and principles please check out the Literature given here for details:
https://docs.open-systems-pharmacology.org/mechanistic-modeling-of-pharmacokinetics-and-dynamics/modeling-concepts/principles-of-pbpk-modeling#introduction-to-pbpk-modeling
Hope this might help you.

Best, Tobias

**Comment by Yuri05** [comment]
As Tobias said, the models are based on ODEs, with the amounts of different molecules (drugs, proteins, ...) serving as state variables.
You can also read [this Wikipedia article on PBPK modeling](https://en.wikipedia.org/wiki/Physiologically_based_pharmacokinetic_modelling), which explains the principle very nicely
